gradle-plugin/src/main/kotlin/io/spine/protodata/gradle/plugin/Plugin.kt

```diff
@@ -281,67 +284,133 @@ private fun Project.configureProtoTask(task: GenerateProtoTask, ext: Extension)
 }
 
 /**
- * Verifies if the project has `java` plugin or `compileKotlin` or `compileTestKotlin` tasks.
+ * Tells if this project may deal with Java code.
  *
- * The current Protobuf support of Kotlin is based on Java codegen. Therefore,
- * it's likely that Java would be enabled in the project for Kotlin proto
- * code to be generated. Though, it may change someday, and Kotlin support for Protobuf would be
+ * @return `true` if `java` plugin is installed, `false` otherwise.
+ */
+private fun Project.hasJava(): Boolean =
+    pluginManager.hasPlugin("java")
+
+/**
+ * Tells if this project may deal with Kotlin code.
```

Comment:
I would say "can" (like below), or "is able". 

"May" is not about a capability, which we characterize here, it's rather about a permission, or a probability.